crapo_workflow/models/workflow_context.py

```diff
@@ -193,10 +194,13 @@ def write(self, values):
         if values.get("done"):
             for wf_context_id in self.mapped("wf_context_id"):
                 filtered_rec = self.filtered(
-                    lambda rec: rec.wf_context_id
+                    lambda rec: rec.exists()
+                    and rec.wf_context_id
```

Comment:
exists() renvoi un recordset filtré et non pas un boolean. Meme si le rec est toujours un recordset de taille 1 (ce qui rend ton code fonctionnel) je prefererais ca : 
```suggestion
                    lambda rec: rec.exists().wf_context_id
```
Si exists() renvoit un recordset vide wf_context_id sera a False

Reply:
Corrigé